src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -183,7 +185,7 @@ export const AddressModal: React.FC<Props> = ({
           action: () => {
             setShowDialog(false)
             closeModal()
-            onDeleteAddress(address.internalID)
+            onDeleteAddress(address?.internalID)
```

Comment:
non-blocking, info: any reason to use safe accessor here vs failing loudly?

Reply:
Since the delete button visible only for the address update, `address` and `internalID` can't be falsy. Updated code to verify `internalID` value before call `onDeleteAddress`